app/forms/webauthn_visit_form.rb

```diff
@@ -66,7 +66,13 @@ def translate_error(error)
     when NOT_SUPPORTED_ERROR
       I18n.t('errors.webauthn_setup.not_supported')
     else
-      I18n.t('errors.webauthn_setup.general_error')
+      I18n.t(
+        'errors.webauthn_setup.general_error_html',
+        link_html: link_to(
+          I18n.t('errors.webauthn_setup.additional_methods_link'),
+          authentication_methods_setup_path,
```

Comment:
In the equivalent changes for Face/Touch in [LG-10228](https://cm-jira.usa.gov/browse/LG-10228), the "choose another authentication method" text was not linked when the user tries adding a new MFA from their account dashboard, because it's not possible to reach to MFA selection page. It was implemented in #8775 as unlinked text when outside account creation. I assume we would want to do the same here.

![image](https://github.com/18F/identity-idp/assets/1779930/86529bcf-cb3f-42ff-84e3-9c8ac2555fa8)

Reply:
@aduth Added your suggestion then reverted. On the same page (form) there's a link that says "‹ Choose another authentication method" which links to authentication_methods_setup. Including the link at the top of the page as well seems consistent.

Maybe there's some confusion over which screen/route. I'm looking at `/webauthn_setup`.

Reply:
Thanks. Returned to this at night and noticed another issue (elsewhere). Will follow with more information.